### ISSUE TITLE
Building 4.2.0 Error with Python3

### ISSUE THREAD

**Issue by ghost** [nextcloud/bookmarks#1540]
**Describe the bug**
I tried building Bookmarks build but something about Python3 went wrong.
It's about ``print()`` changes between Python2 and Python3.

**To Reproduce**
1. Install 4.2.0 with occ
2. ``composer install``
3. ``npm install``

**Expected behavior**
Nothing wrong so that I can do ``npm run build``.

**Server (please complete the following information):**
 - OS: Alpine Edge
 - Nextcloud version: 21.0.1
 - Bookmarks app version: 4.2.0
 - Python3 version: 3.9.5

**Additional context**
```
npm ERR! code 1
npm ERR! path /www/nextcloud/apps/bookmarks/node_modules/node-sass
npm ERR! command failed
npm ERR! command sh -c node scripts/build.js
npm ERR! Building: /usr/bin/node /www/nextcloud/apps/bookmarks/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp verb cli [
npm ERR! gyp verb cli   '/usr/bin/node',
npm ERR! gyp verb cli   '/www/nextcloud/apps/bookmarks/node_modules/node-gyp/bin/node-gyp.js',
npm ERR! gyp verb cli   'rebuild',
npm ERR! gyp verb cli   '--verbose',
npm ERR! gyp verb cli   '--libsass_ext=',
npm ERR! gyp verb cli   '--libsass_cflags=',
npm ERR! gyp verb cli   '--libsass_ldflags=',
npm ERR! gyp verb cli   '--libsass_library='
npm ERR! gyp verb cli ]
npm ERR! gyp info using node-gyp@3.8.0
npm ERR! gyp info using node@14.16.1 | linux | arm
npm ERR! gyp verb command rebuild []
npm ERR! gyp verb command clean []
npm ERR! gyp verb clean removing "build" directory
npm ERR! gyp verb command configure []
npm ERR! gyp verb check python checking for Python executable "/usr/bin/python3" in the PATH
npm ERR! gyp verb `which` succeeded /usr/bin/python3 /usr/bin/python3
npm ERR! gyp ERR! configure error 
npm ERR! gyp ERR! stack Error: Command failed: /usr/bin/python3 -c import sys; print "%s.%s.%s" % sys.version_info[:3];
npm ERR! gyp ERR! stack   File "<string>", line 1
npm ERR! gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
npm ERR! gyp ERR! stack                       ^
npm ERR! gyp ERR! stack SyntaxError: invalid syntax
npm ERR! gyp ERR! stack 
npm ERR! gyp ERR! stack     at ChildProcess.exithandler (child_process.js:308:12)
npm ERR! gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
npm ERR! gyp ERR! stack     at maybeClose (internal/child_process.js:1048:16)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
npm ERR! gyp ERR! System Linux 5.11.10-postmarketos-qcom-msm8974
npm ERR! gyp ERR! command "/usr/bin/node" "/www/nextcloud/apps/bookmarks/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
npm ERR! gyp ERR! cwd /www/nextcloud/apps/bookmarks/node_modules/node-sass
npm ERR! gyp ERR! node -v v14.16.1
npm ERR! gyp ERR! node-gyp -v v3.8.0
npm ERR! gyp ERR! not ok 
npm ERR! Build failed with error code: 1

npm ERR! A complete log of this run can be found in:
npm ERR!     /var/lib/nginx/.npm/_logs/2021-05-13T15_29_12_488Z-debug.log
```

Instead of
```python
import sys
print '%s.%s.%s' % sys.version_info[:3]
```
, Python3 should use
```python
import sys
print('%s.%s.%s' % sys.version_info[:3]);
```

**Comment by HouraisanNEET** [comment]
The problem is fix after node-gyp 4.0.0 while 3.8.0 is not applied.

**Comment by HouraisanNEET** [comment]
I hopped to Alpine 3.13 with Python 3.8.10. Somehow it cannot be reproduced. It's not a problem to me for now.
